action/protocol/execution/protocol_test.go

```diff
@@ -176,9 +182,10 @@ func (cfg *ExecutionConfig) ExpectedReturnValue() []byte {
 
 type SmartContractTest struct {
 	// the order matters
-	InitBalances []ExpectedBalance `json:"initBalances"`
-	Deployments  []ExecutionConfig `json:"deployments"`
-	Executions   []ExecutionConfig `json:"executions"`
+	InitGenesis  GenesisBlockHeight `json:"initGenesis"`
```

Comment:
directly use IsBering bool `json:"isBering"` here?
so no need to define the GenesisBlockHeight struct above